app/common/Kconfig

```diff
@@ -45,6 +45,14 @@ config APP_HEARTBEAT
 	default y
 	depends on GPIO
 
+config APP_LED_ROLLOVER
```

Comment:
Please all `depends on !APP_LED_ROLLOVER` to the `APP_HEARTBEAT` config. This will automatically disable heartbeat when this app is enabled.